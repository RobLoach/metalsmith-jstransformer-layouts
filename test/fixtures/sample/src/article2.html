---
layout: layouts/_page.jade
title: Page 2
---
This is page 2!
