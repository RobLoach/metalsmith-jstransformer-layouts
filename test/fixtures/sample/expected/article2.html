<!DOCTYPE html>
<html>
  <body>
<h1>Page 2</h1>
<article>This is page 2!
</article></body>
</html>